pkg/config/config_test.go

```diff
@@ -155,6 +157,18 @@ func TestNewConfig(t *testing.T) {
 	})
 }
 
+func Test_LoadRemoteConfig(t *testing.T) {
+	t.Run("load-config-valid-url", func(t *testing.T) {
+		c, err := loadRemoteConfig("https://raw.githubusercontent.com/get-woke/woke/main/example.yaml")
+		assert.NoError(t, err)
+		assert.NotNil(t, c)
+	})
+
+	t.Run("load-config-invalid-url", func(t *testing.T) {})
+	_, err := loadRemoteConfig("https://raw.githubusercontent.com/get-woke/woke/main/example")
+	assert.Error(t, err)
```

Comment:
It doesn't look like you're running these in the testing function
```suggestion
	t.Run("load-config-invalid-url", func(t *testing.T) {
		_, err := loadRemoteConfig("https://raw.githubusercontent.com/get-woke/woke/main/example")
		assert.Error(t, err)
	})
```